src/structures/Message.js

```diff
@@ -436,7 +436,7 @@ class Message extends Base {
         await this.client.pupPage.evaluate((msgId, everyone) => {
             let msg = window.Store.Msg.get(msgId);
 
-            if (everyone && msg.id.fromMe && msg._canRevoke()) {
+            if (everyone && msg._canRevoke()) {
                 return window.Store.Cmd.sendRevokeMsgs(msg.chat, [msg], {type: 'Sender'});
```

Comment:
Change the type between "Sender" and "Admin" depending is `msg.id.fromMe`
`return window.Store.Cmd.sendRevokeMsgs(msg.chat, [msg], {type: msg.id.fromMe ? 'Sender' : 'Admin'});`

Reply:
![Captura de pantalla 2022-09-11 145755](https://user-images.githubusercontent.com/99382729/189546663-db0b7a93-83d6-43ba-bc75-139c029e11f5.png)
Works!

Reply:
> Change the type between "Sender" and "Admin" depending is `msg.id.fromMe` `return window.Store.Cmd.sendRevokeMsgs(msg.chat, [msg], {type: msg.id.fromMe ? 'Sender' : 'Admin'});`

Alright